src/main/cpp/auton/drivePrimitives/DrivePathPlanner.h

```diff
@@ -35,18 +35,19 @@
 #include "pathplanner/lib/config/RobotConfig.h"
 #include "pathplanner/lib/trajectory/PathPlannerTrajectory.h"
 
-class DrivePathPlanner : public IPrimitive
+class DrivePathPlanner : public IPrimitive, public DragonDataLogger
 {
 public:
     DrivePathPlanner();
-    ~DrivePathPlanner() = default;
+    ~DrivePathPlanner();
```

Comment:
You can leave the = default; 

If you remove that, then you should implement the destructor in the .cpp file